pkg/platform/baremetal/template.go

```diff
@@ -88,6 +88,21 @@ module "bare-metal-{{.ClusterName}}" {
   download_protocol = "{{ .DownloadProtocol }}"
 
   network_ip_autodetection_method = "{{ .NetworkIPAutodetectionMethod }}"
+
+  {{- if .CLCSnippets}}
+  clc_snippets = {
+    {{- range $nodeName, $clcSnippetList := .CLCSnippets }}
+    "{{ $nodeName }}" = [
+    {{- range $clcSnippet := $clcSnippetList }}
+      <<EOF
+{{ $clcSnippet }}
```

Comment:
Can you please lint?

Reply:
I have thats what terraform fmt suggested.

Reply:
Cool